gc/base/standard/Scavenger.cpp

```diff
@@ -1641,12 +1642,28 @@ MM_Scavenger::updateCopyScanCounts(MM_EnvironmentBase* env, uint64_t slotsScanne
 {
 	env->_scavengerStats._slotsScanned += slotsScanned;
 	env->_scavengerStats._slotsCopied += slotsCopied;
-	uint64_t updateResult = _extensions->copyScanRatio.update(env, &(env->_scavengerStats._slotsScanned), &(env->_scavengerStats._slotsCopied), _waitingCount);
+	uint64_t updateResult = _extensions->copyScanRatio.update(env, &(env->_scavengerStats._slotsScanned), &(env->_scavengerStats._slotsCopied), _waitingCount, _cachedEntryCount, _scavengeCacheScanList.getApproximateEntryCount(), &(env->_scavengerStats._copyScanUpdates));
```

Comment:
_scavengeCacheScanList.getApproximateEntryCount() is not very cheap call. I dont like calling it on minor update.